skill-Collections-Demo-DNN-SMS-to-Account-Phone-Number.json

```diff
@@ -0,0 +1,4496 @@
+{
```

Comment:
@GailMarieDB , can you move this to the `conversation/` directory or is there a reason to not have it inside that directory?

Reply:
Sorry - yes, I can move it.